tanner/emulators/template_injection.py

```diff
@@ -0,0 +1,62 @@
+import asyncio
+import logging
+import os
+
+from urllib.parse import unquote
+from tanner.utils import patterns
+from tanner.config import TannerConfig
+from tanner.utils.aiodocker_helper import AIODockerHelper
+
+
+class TemplateInjection:
+
+    def __init__(self, loop=None):
+        self._loop = loop if loop is not None else asyncio.get_event_loop()
+        self.logger = logging.getLogger('tanner.template_injection')
+        self.docker_helper = AIODockerHelper()
+        self.remote_path = TannerConfig.get('REMOTE_DOCKERFILE', 'GITHUB')
+
+    async def get_injection_result(self, payload):
+        execute_result = None
+        work_dir = os.getcwd()
+
+        # Build the custom image
+        await self.docker_helper.setup_host_image(
+            remote_path=self.remote_path, tag='template_injection:latest')
+
+        if patterns.TEMPLATE_INJECTION_TORNADO.match(payload):
+
+            with open(work_dir + '/tanner/files/engines/tornado.py', 'r') as f:
```

Comment:
tanner can be started at any directory, the file might be not available, you can copy it into the data folder during installation and use the path from config mayb

Reply:
Perfect, made the changes but can you please have a look at this Travis error log?